GeminiApp/VisionAPI_Demo.py

```diff
@@ -0,0 +1,8 @@
+import os
+import io
+from google.cloud import vision
+from google.cloud.vision import types
```

Comment:
I think the issue has to do with where I initiated the venv.. Thinking of retrying it using Visual Studio instead of Pycharm